flixel/text/FlxBitmapText.hx

```diff
@@ -52,6 +52,7 @@ class FlxBitmapText extends FlxSprite
 	/**
 	 * Specifies how the text field should align text.
 	 * JUSTIFY alignment isn't supported.
+	 * Note: For alignment to work, 'autoSize' must be set to false.
```

Comment:
Since autoSize is the default i think this note is a good way to save the user from some tries until he/she figures it out. I can put this note in FlxText also if people agree.